README.md

```diff
@@ -49,7 +49,14 @@ uvicorn app.main:app --reload
 ```
 ### Running tests
 ```shell
-python -m pytest --cov-report term-missing --cov=app tests
+# Standard tests: 'coverage' and 'flake8'
+tox
+# Only flake8 tests
+tox -e flake8
+# Coverage withe html reports
+tox -e rep
+# Marked function tests
+pytest -m MARKER_NAME
```

Comment:
Leave the whole pytest line as it was before :)

Reply:
i hope adding is ok..